ns1/rest/alerts.py

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2024 NSONE, Inc.
+#
+# License under The MIT License (MIT). See LICENSE in project root.
+#
+from . import resource
+
+
+class Alerts(resource.BaseResource):
+    ROOT = "../alerting/v1beta1/alerts"
+    PASSTHRU_FIELDS = [
```

Comment:
what about type/subtype?

Reply:
Those aren't passthru as they are required on create, so can be named params in that call, and are immutable afterwards so aren't specified later.